content/christine_belzie/create-a-url-using-slugs.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: "Create a URL Using Slugs"
+Description: "Step-by-Step guide on creating a slug with URL ."
+DatePublished: "2022-01-14"
+Categories:
+  - "web-development"
+Tags:
+  - "World Wide Web"
+  - "Link"
+CatalogContent:
+  - "introduction-to-the-web"
+  - "paths/web-development"
+---
+
+## Introduction 
+
+Picture this, you're scrolling through the internet, find a cookie recipe, and click on the Share button to send this link to your friend:
+(https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/) 
+
+You see `gluten-free-chocolate-chip-cookies` after the backblash (“/”)? This is called a **slug**. It is a part in the URL that identifies a specific page in a website to make easier for people to read. In other words, slugs explain the content on the page. In this article, you will learn the importance of slugs in your SEO strategy, how to create them, and best practices.   
+
+
+## Why Are Slugs Important to SEO Strategy?
+
+Even though slugs won't have a direct impact on how your content ranks on the web, they do influence a visitor's decision on whether to view your content or not. Here's how:
+
+
+1. **Increases Search Ranking:** Cannibalization is a common issue for content creators. It's when most of the blog posts, articles, and other pieces of content on a website share the same keyword. This makes it difficult for people to find them on your site. Slugs solve this issue by putting a unique name like a title or heading, thus, making it easier for people to find a person's work. Think of this as the names of the folders on your Google Drive or OneDrive account. 
+
+2.  **Improves Shareability:** 
+Adding slugs to your content's urls make them shorter, thus, making it easier for your viewers to copy, paste, and send them to other people. 
+Now, let's create a slug! 
+
+## How to Create a Slug 
+
+Consider using the following methods to creating a slug:
+
+### Method 1: Manually 
+
+1. **Think of a short yet descriptive name:**  We recommend making it easy for people to remember. For example,  let's say a person wanted to create a slug for the blog post titled "How to Make Your Issue Requests Come True".
+
+2. **Place hyphens ("-") in between the parts of your name:** Doing this in the slug would make it easier for users to access the link as they would have to use a CAPS Lock to view the content.  Here's how it would look with the blog post referenced in the previous step: `how-to-make-your-issues-requests-come-true`.
```

Comment:
"Doing this in the slug would make it easier for users to access the link as they would have to use a CAPS Lock to view the content."

I'm not sure what this sentence is referring to? What do hyphens have to do with caps lock?

Reply:
It refers to how to make a URL slug SEO-friendly.

Reply:
@CBID2 I'm sorry, but I don't understand what hyphens have to do with forcing people to use the caps lock key.

Reply:
To clarify, when the hyphens are not used, the user has to use CAPS Lock in order to access a link.

Reply:
Do you mean the shift key?

Reply:
> Do you mean the shift key?

Apparently, it's a case-sensitive issue and readability [issue](https://markitors.com/what-is-a-url-slug/) not so much of an unclickable problem.

Reply:
Ok, but that's not to do with hyphens, that's why slugs use lower case, so the user doesn't need the shift key. The hyphens are because you cannot use a space in a URL, you have to replace it with "%20" which makes things really hard to read.

Reply:
Thanks for clarifying